templates/locales/en.yaml

```diff
@@ -35,18 +35,13 @@ CareTeamInvitationSubject: "Patient invitation"
 CareTeamInviteHeadline: "{{ .PatientName }} wants to share their diabetes data with you on YourLoops."
 CareTeamInviteBody: "YourLoops is a visualization platform for diabetes management and reporting. You’ll find all the data captured by a DBL displayed into charts and graphs."
 CareTeamInviteBody2: "{{ .PatientName }} chose to share their data with you. To respond, click on the link below."
-CareTeamInviteJoin: "Respond to invitation"
```

Comment:
I still see some references to CareTeamInviteJoin in 
- templates/locales/*.yaml (This may be updated through Cowdin ?, and all deleted keys are still in those files)
- templates/source/careteam_invitation.html (don't understand where it is used, it's the only key still present in that folder)

Reply:
locale files will be updated from crowdin.
I fixed the second point